config/standard/contact-summary.templated.js

```diff
@@ -142,27 +156,27 @@ cards = [
       var fields = [];
       var relevantDelivery, birthdate, relevantVisitsANC, relevantVisitsPNC, visitsANC, visitsPNC, subsequentDeliveries, subsequentPregnancies, nextPregnancy;
       reports.forEach(function (report) {
-        if (isReportValid(report) && pregnancyForms.indexOf(report.form) >= 0) {
+        if (isReportValid(report) && extras.pregnancyForms.indexOf(report.form) >= 0) {
 
           // Ignore pregnancies with no delivery report
-          subsequentDeliveries = getSubsequentDeliveries(report);
+          subsequentDeliveries = extras.getSubsequentDeliveries(report);
           if (subsequentDeliveries.length === 0) { return; }
 
-          relevantDelivery = getOldestReport(subsequentDeliveries);
+          relevantDelivery = extras.extras.getOldestReport(subsequentDeliveries);
```

Comment:
Is `extras.extras` correct? Believe it's causing an error

Reply:
Nope! That's a bug for sure.